docker-php-ext-install opcache fails with latest images

**To reproduce**:

`docker pull php:7.0-alpine`
`docker run -it --rm php:7.0-alpine docker-php-ext-install opcache`

**Expected behavior**:

opcache is installed successfully

**Actual behavior**:
```
fetch http://dl-cdn.alpinelinux.org/alpine/v3.4/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.4/community/x86_64/APKINDEX.tar.gz
(1/27) Installing m4 (1.4.17-r1)
(2/27) Installing perl (5.22.2-r0)
(3/27) Installing autoconf (2.69-r0)
(4/27) Installing pkgconf (0.9.12-r0)
(5/27) Installing pkgconfig (0.25-r1)
(6/27) Installing dpkg-dev (1.18.7-r0)
(7/27) Installing libbz2 (1.0.6-r5)
(8/27) Installing dpkg (1.18.7-r0)
(9/27) Installing libmagic (5.27-r0)
(10/27) Installing file (5.27-r0)
(11/27) Installing libgcc (5.3.0-r0)
(12/27) Installing libstdc++ (5.3.0-r0)
(13/27) Installing binutils-libs (2.26-r1)
(14/27) Installing binutils (2.26-r1)
(15/27) Installing gmp (6.1.0-r0)
(16/27) Installing isl (0.14.1-r0)
(17/27) Installing libgomp (5.3.0-r0)
(18/27) Installing libatomic (5.3.0-r0)
(19/27) Installing mpfr3 (3.1.2-r0)
(20/27) Installing mpc1 (1.0.3-r0)
(21/27) Installing gcc (5.3.0-r0)
(22/27) Installing musl-dev (1.1.14-r14)
(23/27) Installing libc-dev (0.7-r0)
(24/27) Installing g++ (5.3.0-r0)
(25/27) Installing make (4.1-r1)
(26/27) Installing re2c (0.14.3-r0)
(27/27) Installing .phpize-deps (0)
Executing busybox-1.24.2-r13.trigger
OK: 220 MiB in 53 packages
+ cd opcache
+ phpize
Configuring for:
PHP Api Version:         20151012
Zend Module Api No:      20151012
Zend Extension Api No:   320151012
+ ./configure --build=x86_64-linux-musl
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for a sed that does not truncate output... /bin/sed
expr: syntax error
sh: 0: unknown operand
expr: syntax error
sh: 0: unknown operand
expr: syntax error
sh: 0: unknown operand
expr: syntax error
sh: 0: unknown operand
expr: syntax error
sh: 0: unknown operand
checking for cc... cc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether cc accepts -g... yes
checking for cc option to accept ISO C89... none needed
checking how to run the C preprocessor... cc -E
checking for icc... no
checking for suncc... no
checking whether cc understands -c and -o together... yes
checking for system library directory... lib
checking if compiler supports -R... no
checking if compiler supports -Wl,-rpath,... yes
checking build system type... x86_64-pc-linux-musl
checking host system type... x86_64-pc-linux-musl
checking target system type... x86_64-pc-linux-musl
checking for PHP prefix... /usr/local
checking for PHP includes... -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib
checking for PHP extension directory... /usr/local/lib/php/extensions/no-debug-non-zts-20151012
checking for PHP installed headers prefix... /usr/local/include/php
checking if debug is enabled... no
checking if zts is enabled... no
checking for re2c... re2c
checking for re2c version... 0.14.3 (ok)
checking for gawk... no
checking for nawk... no
checking for awk... awk
checking if awk is broken... no
checking whether to enable Zend OPcache support... yes, shared
checking whether to enable file based caching... yes
checking whether to enable copying PHP CODE pages into HUGE PAGES... yes
checking for mprotect... yes
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for unistd.h... (cached) yes
checking sys/uio.h usability... yes
checking sys/uio.h presence... yes
checking for sys/uio.h... yes
checking for sysvipc shared memory support... yes
checking for mmap() using MAP_ANON shared memory support... yes
checking for mmap() using /dev/zero shared memory support... yes
checking for mmap() using shm_open() shared memory support... yes
checking for mmap() using regular file shared memory support... yes
checking "whether flock struct is linux ordered"... "yes"
checking "whether flock struct is BSD ordered"... "no"
checking for ld used by cc... /usr/x86_64-alpine-linux-musl/bin/ld
checking if the linker (/usr/x86_64-alpine-linux-musl/bin/ld) is GNU ld... yes
checking for /usr/x86_64-alpine-linux-musl/bin/ld option to reload object files... -r
checking for BSD-compatible nm... /usr/bin/nm -B
checking whether ln -s works... yes
checking how to recognize dependent libraries... pass_all
checking dlfcn.h usability... yes
checking dlfcn.h presence... yes
checking for dlfcn.h... yes
checking the maximum length of command line arguments... 98304
checking command to parse /usr/bin/nm -B output from cc object... ok
checking for objdir... .libs
checking for ar... ar
checking for ranlib... ranlib
checking for strip... strip
checking if cc supports -fno-rtti -fno-exceptions... no
checking for cc option to produce PIC... -fPIC
checking if cc PIC flag -fPIC works... yes
checking if cc static flag -static works... yes
checking if cc supports -c -o file.o... yes
checking whether the cc linker (/usr/x86_64-alpine-linux-musl/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no

creating libtool
appending configuration tag "CXX" to libtool
configure: creating ./config.status
config.status: creating config.h
/bin/sh /usr/src/php/ext/opcache/libtool --mode=compile cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/usr/src/php/ext/opcache -DPHP_ATOM_INC -I/usr/src/php/ext/opcache/include -I/usr/src/php/ext/opcache/main -I/usr/src/php/ext/opcache -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib  -fstack-protector-strong -fpic -fpie -O2 -DHAVE_CONFIG_H  -fstack-protector-strong -fpic -fpie -O2   -c /usr/src/php/ext/opcache/ZendAccelerator.c -o ZendAccelerator.lo
mkdir .libs
 cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/usr/src/php/ext/opcache -DPHP_ATOM_INC -I/usr/src/php/ext/opcache/include -I/usr/src/php/ext/opcache/main -I/usr/src/php/ext/opcache -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -fstack-protector-strong -fpic -fpie -O2 -DHAVE_CONFIG_H -fstack-protector-strong -fpic -fpie -O2 -c /usr/src/php/ext/opcache/ZendAccelerator.c  -fPIC -DPIC -o .libs/ZendAccelerator.o
In file included from /usr/src/php/ext/opcache/ZendAccelerator.c:42:0:
/usr/local/include/php/ext/pcre/php_pcre.h:29:18: fatal error: pcre.h: No such file or directory
compilation terminated.
Makefile:193: recipe for target 'ZendAccelerator.lo' failed
make: *** [ZendAccelerator.lo] Error 1
```

Same result with 7.1 image.

Note:
Output of:
`docker run -it --rm php:7.0-alpine php --version`
is:
```
PHP 7.0.18 (cli) (built: May 11 2017 18:43:29) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2017 Zend Technologies
```

So, it is not because of the update to 7.0.19 (which is not in docker repos as of now).

--- a/5.6/Dockerfile
+++ b/5.6/Dockerfile
@@ -14,9 +14,9 @@ ENV PHPIZE_DEPS \
 		g++ \
 		gcc \
 		libc-dev \
+		libpcre3-dev \
 		make \
 		pkg-config \
-		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \

--- a/5.6/Dockerfile
+++ b/5.6/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkg-config \
+		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \
@@ -89,7 +90,6 @@ RUN set -xe \
 		$PHP_EXTRA_BUILD_DEPS \
 		libcurl4-openssl-dev \
 		libedit-dev \
-		libpcre3-dev \
 		libsqlite3-dev \
 		libssl-dev \
 		libxml2-dev \

--- a/5.6/alpine/Dockerfile
+++ b/5.6/alpine/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkgconf \
+		pcre-dev \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \
@@ -95,7 +96,6 @@ RUN set -xe \
 		libedit-dev \
 		libxml2-dev \
 		openssl-dev \
-		pcre-dev \
 		sqlite-dev \
 	\
 	&& export CFLAGS="$PHP_CFLAGS" \

--- a/5.6/alpine/Dockerfile
+++ b/5.6/alpine/Dockerfile
@@ -15,8 +15,8 @@ ENV PHPIZE_DEPS \
 		gcc \
 		libc-dev \
 		make \
-		pkgconf \
 		pcre-dev \
+		pkgconf \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \

--- a/5.6/apache/Dockerfile
+++ b/5.6/apache/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkg-config \
+		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \
@@ -148,7 +149,6 @@ RUN set -xe \
 		$PHP_EXTRA_BUILD_DEPS \
 		libcurl4-openssl-dev \
 		libedit-dev \
-		libpcre3-dev \
 		libsqlite3-dev \
 		libssl-dev \
 		libxml2-dev \

--- a/5.6/apache/Dockerfile
+++ b/5.6/apache/Dockerfile
@@ -14,9 +14,9 @@ ENV PHPIZE_DEPS \
 		g++ \
 		gcc \
 		libc-dev \
+		libpcre3-dev \
 		make \
 		pkg-config \
-		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \

--- a/5.6/fpm/Dockerfile
+++ b/5.6/fpm/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkg-config \
+		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \
@@ -90,7 +91,6 @@ RUN set -xe \
 		$PHP_EXTRA_BUILD_DEPS \
 		libcurl4-openssl-dev \
 		libedit-dev \
-		libpcre3-dev \
 		libsqlite3-dev \
 		libssl-dev \
 		libxml2-dev \

--- a/5.6/fpm/Dockerfile
+++ b/5.6/fpm/Dockerfile
@@ -14,9 +14,9 @@ ENV PHPIZE_DEPS \
 		g++ \
 		gcc \
 		libc-dev \
+		libpcre3-dev \
 		make \
 		pkg-config \
-		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \

--- a/5.6/fpm/alpine/Dockerfile
+++ b/5.6/fpm/alpine/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkgconf \
+		pcre-dev \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \
@@ -96,7 +97,6 @@ RUN set -xe \
 		libedit-dev \
 		libxml2-dev \
 		openssl-dev \
-		pcre-dev \
 		sqlite-dev \
 	\
 	&& export CFLAGS="$PHP_CFLAGS" \

--- a/5.6/fpm/alpine/Dockerfile
+++ b/5.6/fpm/alpine/Dockerfile
@@ -15,8 +15,8 @@ ENV PHPIZE_DEPS \
 		gcc \
 		libc-dev \
 		make \
-		pkgconf \
 		pcre-dev \
+		pkgconf \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \

--- a/5.6/zts/Dockerfile
+++ b/5.6/zts/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkg-config \
+		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \
@@ -90,7 +91,6 @@ RUN set -xe \
 		$PHP_EXTRA_BUILD_DEPS \
 		libcurl4-openssl-dev \
 		libedit-dev \
-		libpcre3-dev \
 		libsqlite3-dev \
 		libssl-dev \
 		libxml2-dev \

--- a/5.6/zts/Dockerfile
+++ b/5.6/zts/Dockerfile
@@ -14,9 +14,9 @@ ENV PHPIZE_DEPS \
 		g++ \
 		gcc \
 		libc-dev \
+		libpcre3-dev \
 		make \
 		pkg-config \
-		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \

--- a/5.6/zts/alpine/Dockerfile
+++ b/5.6/zts/alpine/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkgconf \
+		pcre-dev \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \
@@ -96,7 +97,6 @@ RUN set -xe \
 		libedit-dev \
 		libxml2-dev \
 		openssl-dev \
-		pcre-dev \
 		sqlite-dev \
 	\
 	&& export CFLAGS="$PHP_CFLAGS" \

--- a/5.6/zts/alpine/Dockerfile
+++ b/5.6/zts/alpine/Dockerfile
@@ -15,8 +15,8 @@ ENV PHPIZE_DEPS \
 		gcc \
 		libc-dev \
 		make \
-		pkgconf \
 		pcre-dev \
+		pkgconf \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \

--- a/7.0/Dockerfile
+++ b/7.0/Dockerfile
@@ -14,9 +14,9 @@ ENV PHPIZE_DEPS \
 		g++ \
 		gcc \
 		libc-dev \
+		libpcre3-dev \
 		make \
 		pkg-config \
-		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \

--- a/7.0/Dockerfile
+++ b/7.0/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkg-config \
+		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \
@@ -89,7 +90,6 @@ RUN set -xe \
 		$PHP_EXTRA_BUILD_DEPS \
 		libcurl4-openssl-dev \
 		libedit-dev \
-		libpcre3-dev \
 		libsqlite3-dev \
 		libssl-dev \
 		libxml2-dev \

--- a/7.0/alpine/Dockerfile
+++ b/7.0/alpine/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkgconf \
+		pcre-dev \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \
@@ -95,7 +96,6 @@ RUN set -xe \
 		libedit-dev \
 		libxml2-dev \
 		openssl-dev \
-		pcre-dev \
 		sqlite-dev \
 	\
 	&& export CFLAGS="$PHP_CFLAGS" \

--- a/7.0/alpine/Dockerfile
+++ b/7.0/alpine/Dockerfile
@@ -15,8 +15,8 @@ ENV PHPIZE_DEPS \
 		gcc \
 		libc-dev \
 		make \
-		pkgconf \
 		pcre-dev \
+		pkgconf \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \

--- a/7.0/apache/Dockerfile
+++ b/7.0/apache/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkg-config \
+		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \
@@ -148,7 +149,6 @@ RUN set -xe \
 		$PHP_EXTRA_BUILD_DEPS \
 		libcurl4-openssl-dev \
 		libedit-dev \
-		libpcre3-dev \
 		libsqlite3-dev \
 		libssl-dev \
 		libxml2-dev \

--- a/7.0/apache/Dockerfile
+++ b/7.0/apache/Dockerfile
@@ -14,9 +14,9 @@ ENV PHPIZE_DEPS \
 		g++ \
 		gcc \
 		libc-dev \
+		libpcre3-dev \
 		make \
 		pkg-config \
-		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \

--- a/7.0/fpm/Dockerfile
+++ b/7.0/fpm/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkg-config \
+		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \
@@ -90,7 +91,6 @@ RUN set -xe \
 		$PHP_EXTRA_BUILD_DEPS \
 		libcurl4-openssl-dev \
 		libedit-dev \
-		libpcre3-dev \
 		libsqlite3-dev \
 		libssl-dev \
 		libxml2-dev \

--- a/7.0/fpm/Dockerfile
+++ b/7.0/fpm/Dockerfile
@@ -14,9 +14,9 @@ ENV PHPIZE_DEPS \
 		g++ \
 		gcc \
 		libc-dev \
+		libpcre3-dev \
 		make \
 		pkg-config \
-		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \

--- a/7.0/fpm/alpine/Dockerfile
+++ b/7.0/fpm/alpine/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkgconf \
+		pcre-dev \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \
@@ -96,7 +97,6 @@ RUN set -xe \
 		libedit-dev \
 		libxml2-dev \
 		openssl-dev \
-		pcre-dev \
 		sqlite-dev \
 	\
 	&& export CFLAGS="$PHP_CFLAGS" \

--- a/7.0/fpm/alpine/Dockerfile
+++ b/7.0/fpm/alpine/Dockerfile
@@ -15,8 +15,8 @@ ENV PHPIZE_DEPS \
 		gcc \
 		libc-dev \
 		make \
-		pkgconf \
 		pcre-dev \
+		pkgconf \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \

--- a/7.0/zts/Dockerfile
+++ b/7.0/zts/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkg-config \
+		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \
@@ -90,7 +91,6 @@ RUN set -xe \
 		$PHP_EXTRA_BUILD_DEPS \
 		libcurl4-openssl-dev \
 		libedit-dev \
-		libpcre3-dev \
 		libsqlite3-dev \
 		libssl-dev \
 		libxml2-dev \

--- a/7.0/zts/Dockerfile
+++ b/7.0/zts/Dockerfile
@@ -14,9 +14,9 @@ ENV PHPIZE_DEPS \
 		g++ \
 		gcc \
 		libc-dev \
+		libpcre3-dev \
 		make \
 		pkg-config \
-		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \

--- a/7.0/zts/alpine/Dockerfile
+++ b/7.0/zts/alpine/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkgconf \
+		pcre-dev \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \
@@ -96,7 +97,6 @@ RUN set -xe \
 		libedit-dev \
 		libxml2-dev \
 		openssl-dev \
-		pcre-dev \
 		sqlite-dev \
 	\
 	&& export CFLAGS="$PHP_CFLAGS" \

--- a/7.0/zts/alpine/Dockerfile
+++ b/7.0/zts/alpine/Dockerfile
@@ -15,8 +15,8 @@ ENV PHPIZE_DEPS \
 		gcc \
 		libc-dev \
 		make \
-		pkgconf \
 		pcre-dev \
+		pkgconf \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \

--- a/7.1/Dockerfile
+++ b/7.1/Dockerfile
@@ -14,9 +14,9 @@ ENV PHPIZE_DEPS \
 		g++ \
 		gcc \
 		libc-dev \
+		libpcre3-dev \
 		make \
 		pkg-config \
-		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \

--- a/7.1/Dockerfile
+++ b/7.1/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkg-config \
+		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \
@@ -89,7 +90,6 @@ RUN set -xe \
 		$PHP_EXTRA_BUILD_DEPS \
 		libcurl4-openssl-dev \
 		libedit-dev \
-		libpcre3-dev \
 		libsqlite3-dev \
 		libssl-dev \
 		libxml2-dev \

--- a/7.1/alpine/Dockerfile
+++ b/7.1/alpine/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkgconf \
+		pcre-dev \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \
@@ -95,7 +96,6 @@ RUN set -xe \
 		libedit-dev \
 		libxml2-dev \
 		openssl-dev \
-		pcre-dev \
 		sqlite-dev \
 	\
 	&& export CFLAGS="$PHP_CFLAGS" \

--- a/7.1/alpine/Dockerfile
+++ b/7.1/alpine/Dockerfile
@@ -15,8 +15,8 @@ ENV PHPIZE_DEPS \
 		gcc \
 		libc-dev \
 		make \
-		pkgconf \
 		pcre-dev \
+		pkgconf \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \

--- a/7.1/apache/Dockerfile
+++ b/7.1/apache/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkg-config \
+		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \
@@ -148,7 +149,6 @@ RUN set -xe \
 		$PHP_EXTRA_BUILD_DEPS \
 		libcurl4-openssl-dev \
 		libedit-dev \
-		libpcre3-dev \
 		libsqlite3-dev \
 		libssl-dev \
 		libxml2-dev \

--- a/7.1/apache/Dockerfile
+++ b/7.1/apache/Dockerfile
@@ -14,9 +14,9 @@ ENV PHPIZE_DEPS \
 		g++ \
 		gcc \
 		libc-dev \
+		libpcre3-dev \
 		make \
 		pkg-config \
-		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \

--- a/7.1/fpm/Dockerfile
+++ b/7.1/fpm/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkg-config \
+		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \
@@ -90,7 +91,6 @@ RUN set -xe \
 		$PHP_EXTRA_BUILD_DEPS \
 		libcurl4-openssl-dev \
 		libedit-dev \
-		libpcre3-dev \
 		libsqlite3-dev \
 		libssl-dev \
 		libxml2-dev \

--- a/7.1/fpm/Dockerfile
+++ b/7.1/fpm/Dockerfile
@@ -14,9 +14,9 @@ ENV PHPIZE_DEPS \
 		g++ \
 		gcc \
 		libc-dev \
+		libpcre3-dev \
 		make \
 		pkg-config \
-		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \

--- a/7.1/fpm/alpine/Dockerfile
+++ b/7.1/fpm/alpine/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkgconf \
+		pcre-dev \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \
@@ -96,7 +97,6 @@ RUN set -xe \
 		libedit-dev \
 		libxml2-dev \
 		openssl-dev \
-		pcre-dev \
 		sqlite-dev \
 	\
 	&& export CFLAGS="$PHP_CFLAGS" \

--- a/7.1/fpm/alpine/Dockerfile
+++ b/7.1/fpm/alpine/Dockerfile
@@ -15,8 +15,8 @@ ENV PHPIZE_DEPS \
 		gcc \
 		libc-dev \
 		make \
-		pkgconf \
 		pcre-dev \
+		pkgconf \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \

--- a/7.1/zts/Dockerfile
+++ b/7.1/zts/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkg-config \
+		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \
@@ -90,7 +91,6 @@ RUN set -xe \
 		$PHP_EXTRA_BUILD_DEPS \
 		libcurl4-openssl-dev \
 		libedit-dev \
-		libpcre3-dev \
 		libsqlite3-dev \
 		libssl-dev \
 		libxml2-dev \

--- a/7.1/zts/Dockerfile
+++ b/7.1/zts/Dockerfile
@@ -14,9 +14,9 @@ ENV PHPIZE_DEPS \
 		g++ \
 		gcc \
 		libc-dev \
+		libpcre3-dev \
 		make \
 		pkg-config \
-		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \

--- a/7.1/zts/alpine/Dockerfile
+++ b/7.1/zts/alpine/Dockerfile
@@ -16,6 +16,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkgconf \
+		pcre-dev \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \
@@ -96,7 +97,6 @@ RUN set -xe \
 		libedit-dev \
 		libxml2-dev \
 		openssl-dev \
-		pcre-dev \
 		sqlite-dev \
 	\
 	&& export CFLAGS="$PHP_CFLAGS" \

--- a/7.1/zts/alpine/Dockerfile
+++ b/7.1/zts/alpine/Dockerfile
@@ -15,8 +15,8 @@ ENV PHPIZE_DEPS \
 		gcc \
 		libc-dev \
 		make \
-		pkgconf \
 		pcre-dev \
+		pkgconf \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \

--- a/Dockerfile-alpine.template
+++ b/Dockerfile-alpine.template
@@ -10,6 +10,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkgconf \
+		pcre-dev \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \
@@ -89,7 +90,6 @@ RUN set -xe \
 		libedit-dev \
 		libxml2-dev \
 		openssl-dev \
-		pcre-dev \
 		sqlite-dev \
 	\
 	&& export CFLAGS="$PHP_CFLAGS" \

--- a/Dockerfile-alpine.template
+++ b/Dockerfile-alpine.template
@@ -9,8 +9,8 @@ ENV PHPIZE_DEPS \
 		gcc \
 		libc-dev \
 		make \
-		pkgconf \
 		pcre-dev \
+		pkgconf \
 		re2c
 RUN apk add --no-cache --virtual .persistent-deps \
 		ca-certificates \

--- a/Dockerfile-debian.template
+++ b/Dockerfile-debian.template
@@ -10,6 +10,7 @@ ENV PHPIZE_DEPS \
 		libc-dev \
 		make \
 		pkg-config \
+		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \
@@ -83,7 +84,6 @@ RUN set -xe \
 		$PHP_EXTRA_BUILD_DEPS \
 		libcurl4-openssl-dev \
 		libedit-dev \
-		libpcre3-dev \
 		libsqlite3-dev \
 		libssl-dev \
 		libxml2-dev \

--- a/Dockerfile-debian.template
+++ b/Dockerfile-debian.template
@@ -8,9 +8,9 @@ ENV PHPIZE_DEPS \
 		g++ \
 		gcc \
 		libc-dev \
+		libpcre3-dev \
 		make \
 		pkg-config \
-		libpcre3-dev \
 		re2c
 RUN apt-get update && apt-get install -y \
 		$PHPIZE_DEPS \

